docs/book/src/developer/providers/implementers-guide/generate_crds.md

```diff
@@ -37,8 +37,8 @@ Here you will be asked if you want to generate resources and controllers.
 You'll want both of them:
 
 ```bash
-kubebuilder create api --group infrastructure --version v1alpha3 --kind MailgunCluster
-kubebuilder create api --group infrastructure --version v1alpha3 --kind MailgunMachine
+kubebuilder create api --group infrastructure --version v1beta1 --kind MailgunCluster
```

Comment:
```suggestion
kubebuilder create api --group infrastructure --version v1alpha1 --kind MailgunCluster
```
Every provider can have its own API version, no need to stay in sync with CAPI. Given that I think we should suggest everyone to start from v1alpha1

Reply:
Sounds like a good idea to use v1alpha1 for the infra provider apiVersion across the guide to make clear it doesn't have to be the same version as core API or other providers are using.

Reply:
done